docs/user/profile.rst

```diff
@@ -4,36 +4,40 @@ Registration and user profile
 Registration
 ------------
 
-While by default everybody can browse projects, view translations or suggest them, only
-registered users are allowed to actually save changes and are credited for
+Everybody can browse projects, view translations or suggest translations by default.
+Only registered users are allowed to actually save changes, and are credited for
 every translation made.
 
 You can register by following a few simple steps:
 
-1. Fill out the registration form with your credentials
-2. Activate registration by following the link in the e-mail you receive
-3. Possibly adjust your profile to choose which languages you know
+1. Fill out the registration form with your credentials.
+2. Activate registration by following the link in the e-mail you receive.
+3. Optionally adjust your profile to choose which languages you know.
 
 .. _dashboard:
 
 Dashboard
 ---------
 
-When you log in to Weblate, you will see an overview of projects and components
-as well as their translation progress.
+When you log in, you will see an overview of projects and components,
+as well as their respective translation progression.
 
 .. versionadded:: 2.5
 
-By default, this will show the components of projects you are watching,
-cross-referenced with your preferred languages.  You can switch to different
-views using the navigation tabs.
+Components of projects you are watching are shown by default, and
+cross-referenced with your preferred languages.
+
+.. hint::
+
+You can switch to different views using the navigation tabs.
```

Comment:
```suggestion
    You can switch to different views using the navigation tabs.
```

The `.. hint::` content has to be indented, see https://docutils.sourceforge.io/docs/ref/rst/directives.html#hint

Reply:
@comradekingu can you add what you wanted there?